account_global_discount/models/account_move.py

```diff
@@ -320,8 +320,8 @@ class AccountMoveLine(models.Model):
         string="Amount Untaxed Before Discounts",
         readonly=True,
     )
-    # TODO: To be removed on future versions if invoice_global_discount_id is properly filled
-    # Provided for compatibility in stable branch
+    # TODO: To be removed on future versions if invoice_global_discount_id
+    # is properly filled Provided for compatibility in stable branch
```

Comment:
This is strange, shouldn't be placed this change in previous commit, if it was a `pre-commit` execution consequence?